packages/core/src/utils/security/handlers/bearer.ts

```diff
@@ -18,15 +18,15 @@ function isBearerToken(inputHeaders: Headers) {
 
 export const bearer = {
   test: ({ type, scheme }: SecurityScheme) => scheme === 'bearer' && type === 'http',
-  handle: bearerHandler.bind(undefined, 'Bearer'),
+  handle: partial(bearerHandler, 'Bearer'),
```

Comment:
hm.. why do we care for `this`? it should burn in hell

Reply:
We do not — `partial` will indeed make sure to not touch the `this`; it's basically a nicer `bearerHandler.bind(undefined, 'Bearer')`